### PR TITLE
Add progress bar component

### PR DESCRIPTION
**Pull request by StephenAraka**
### What does this PR do?
create ProgressBar component.

### Screenshots?
Screenshot from 2020-02-24 14-10-16

### Usage...
takes in a prop of percentage
Screenshot from 2020-02-24 14-11-30

🤗...🤗...🤗...🤗

### CODE REVIEW

**Comment by mwotil** [comment]
The colors are sleek!! Keep it up @StephenAraka

File: src/components/ProgressBar/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  return (
+    <div className="ProgressBar">
+      <div
+        className={`ProgressBarFiller ProgressBarFill${fillColor}`}
+        style={{width: `${props.percentage}%`}}
+  >
+    {`${props.percentage}%`}
+    </div>
```
Comment (Mubangizi) [inline src/components/ProgressBar/index.js]: Poor indenting from line 22 to 24

Comment (StephenAraka) [reply src/components/ProgressBar/index.js]: Thanks for pointing this out @Mubangizi ... U got an eagle eye man...👁 
I've fixed it now

Comment (Mubangizi) [reply src/components/ProgressBar/index.js]: Good stuff @StephenAraka

**Pull request merged**